shotover-proxy/src/transforms/cassandra/sink_cluster.rs

```diff
@@ -229,37 +235,50 @@ pub fn create_topology_task(
 async fn topology_task_process(
     tls: &Option<TlsConnector>,
     nodes: &Arc<RwLock<Vec<CassandraNode>>>,
-    handshake: TaskHandshake,
+    handshake: &TaskHandshake,
     data_center: &str,
 ) -> Result<()> {
-    let outbound = new_connection(handshake.address, &handshake.handshake, tls, &None).await?;
+    let outbound = new_connection(&handshake.address, &handshake.handshake, tls, &None).await?;
+
+    let (peers_tx, peers_rx) = oneshot::channel();
+    outbound.send(
+        Message::from_frame(Frame::Cassandra(CassandraFrame {
+            version: Version::V4,
+            stream_id: 1000,
```

Comment:
Why 1000? I thought setting to 0 was more common when you don't care about the stream id.

Reply:
These are cleaned up frames that the driver output and thats where the 1000 value came from.
We could probably do 0 and 1 and to make it look less strange.

Reply:
Hmmm before we land this I need to look into how robust our stream id handling is against repeating stream ids.

Reply:
Oh its fine, in that case the value at stream_id in return_channel_map and return_message_map just gets replaced with the new message.

I've now replaced the stream_id's with 0 and 1.
I also tweaked the async logic a bit so that we can start processing whichever message comes back first.